lib/src/parser/recursive.dart

```diff
@@ -187,6 +187,41 @@ class RecursiveAstParser {
         );
       }
     }
+
+    // At this point, it is a TextAttribute, but handle cases
+    // with 'on-' and 'bind-' prefix.
+    if (decoratorToken.lexeme.startsWith('on-')) {
```

Comment:
Adding 'on-' and 'bind-' to NgSimpleTokenTypes just adds complexity without any extra feature. For all intents and purposes, the syntactical functionality of 'on-' or 'bind-' is no more different than a standard text attribute parsing. 

I can add an error in this logic for if the term after on-/bind- is an empty string.